packages/redux-dynostore-core/src/dynamicReducers.js

```diff
@@ -28,11 +33,18 @@ const dynamicReducersEnhancer = () => createHandlers => (store, reducer, ...rest
     store.replaceReducer(createReducer())
   }
 
+  const detachReducer = identifier => {
```

Comment:
How possible/annoying would it be to make this `detachReducers` and accept an array of identifiers (or `(...identifiers)`) so it matches `attacherReducers`?

I'm just thinking of cases where you want to remove multiple identifiers at once, could become more expensive that batching them together but the cost of passing just one would be comparable to this.

Reply:
I have actioned this by running a forEach, however I think this is far more expensive than it likely needs to be.
```
identifiers.forEach(identifier => {
      store.dispatch(globalAction({ type: DETACH_TYPE, identifier }))
      delete dynamicReducers[identifier]
      store.replaceReducer(createReducer())
})
```

I tried reducing the complexity by doing
```
identifiers.forEach(identifier => {
      store.dispatch(globalAction({ type: DETACH_TYPE, identifier }))
      delete dynamicReducers[identifier]
})
store.replaceReducer(createReducer())
```
But only the last reducer was being removed.

I'm positive there is a more efficient way of achieving this, but I'm not sure how, sorry! :)
Happy to revert back to a single identifier if you'd prefer, or if you have time to help guide me in a better solution I can look into implementing it.